README.md

```diff
@@ -140,6 +140,41 @@ This performs an NVE molecular dynamics simulation at 300K for 200 steps (0.2 ps
 For all options, run `janus md --help`.
 
 
+### Using configuration files
+
+Default values for all command line options may be specifed through a Yaml 1.1 formatted configuration file by adding the `--config` option. If an option is present in both the command line and configuration file, the command line value takes precedence.
+
+For example, with the following configuration file and command:
+
+```yaml
+struct: "NaCl.cif"
+properties:
```

Comment:
? based on another comment is this correct?

Reply:
It's no longer correct (it doesn't work very well defining the path here for tests while keeping it generic and easily removable) but I kept it as something to be overwritten by the explicit `--struct`.

Maybe I should make it more explicitly wrong.

Reply:
Oh sorry I was thinking this is the test config, which bit is the problem?

Reply:
`janus singlepoint --config config.yml` works with this config if `NaCl.cif` is in your current directory, so perhaps the comment is wrong/misleading?